requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
```

Comment:
Are you using Python 3.8 and `pip-tools==6.6.2` to run `pip-compile`?

Reply:
beautiful, thanks xD